src/utils/date-time.js

```diff
@@ -76,6 +76,14 @@ module.exports = {
   getFirstDayOfMonth(d) {
     return new Date(d.getFullYear(), d.getMonth(), 1);
   },
+  
+  getLocalizedWeekdayTitle(weekday, intl, locale, weekdayFormat) {
+    let weekdayFormatter = new intl.DateTimeFormat(locale, { weekday: weekdayFormat });
+    let now = new Date();
+    let firstDayOfWeek = new Date(now.setDate(now.getDate() - now.getDay()));
+    
+    return weekdayFormatter.format(this.addDays(firstDayOfWeek, weekday));
```

Comment:
I think that we could save a call to `this.addDays`
(http://stackoverflow.com/questions/4156434/javascript-get-the-first-day-of-the-week-from-current-date)